internal/syncer/connector/webdav_test.go

```
@@ -170,6 +173,76 @@ func webDAVTestConnector(t *testing.T, serverURL string, allowImages bool, batch
+	if err := os.WriteFile(caCertPath, caPEM, 0o600); err != nil {
+		t.Fatalf("write CA certificate: %v", err)
+	}
+
+	connector, err := NewWebDAVConnector(map[string]any{
+		"base_url":     server.URL,
+		"ca_cert_path": caCertPath,
+	})
+	if err != nil {
+		t.Fatalf("NewWebDAVConnector failed: %v", err)
+	}
+	resp, err := connector.client.httpClient.Get(server.URL)
```

golangci-lint (2.12.2)

[error] (*net/http.Client).Get must not be called. use (*net/http.Client).Do(*http.Request)

(noctx)